patches/aedes+0.50.0.patch

```diff
@@ -42,7 +53,14 @@ index c02d289..0e298ef 100644
  
    this.decodeProtocol = opts.decodeProtocol
    this.trustProxy = opts.trustProxy
-@@ -250,6 +246,15 @@ function removeSharp (sub) {
+   this.trustedProxies = opts.trustedProxies
+ 
++  this._wrapDeliveryFunc = opts.wrapDeliveryFunc
```

Comment:
this is the new function i had to introduce to reliably modify the delivery function passed to `Aedes.subscribe` and `Aedes.unsubscribe` (stored in client.subscriptions map).
In this PR i am patching it to allow modifying the delivery function inside our library, but it can be used directly by the end user, which is a start of enabling a middleware approach (instead of monkey patching).
See [discussion](https://github.com/moscajs/aedes/issues/888#issuecomment-1723021555)

Reply:
It's ok 👍🏼